src/item.js

```diff
@@ -6,62 +6,61 @@ export default class Item {
 		this.url = url;
 		this.notificationSent = false;
 		this.shouldSendNotification = true;
-		this.html = undefined;
+		this.html = null;
 		this.info = {
-			title: undefined,
-			inventory: undefined,
-			image: undefined,
+			title: null,
+			inventory: null,
+			image: null,
```

Comment:
Please tell me whether this makes any difference or the reason this was changed.